packages/parse5-htmlparser2-tree-adapter/package.json

```diff
@@ -14,14 +14,20 @@
         "htmlparser2"
     ],
     "license": "MIT",
-    "main": "dist/index.js",
+    "main": "dist/cjs/index.js",
```

Comment:
are we sure the default entrypoint should be commonjs? shouldn't we be esm first and cjs second? i.e. only reference the cjs in the exports map

Reply:
Here is my thinking: The `main` field is only used by legacy NodeJS (everything else either uses the other fields). These versions don't have ESM support, and giving them anything but CJS wouldn't work.

Reply:
yeah that makes sense. my concern was that tooling that also only looks at `main` would be forced to get CJS. though AFAIK most tooling these days does understand the `module` field or the `exports` field, so we're probably fine 👍